### PR TITLE
More flexible mail templates

### PR DESCRIPTION
**Pull request by eladeyal-intel**
see frab#566

### CODE REVIEW

**Comment by eladeyal-intel** [comment]
Merge note: This is built over #555 because there of the following dependencies:
1. db/schema.rb
2. sending e-mails to "stakeholders" (including assistants)

File: config/locales/de.yml [hunk 1]
```diff
@@ -896,9 +896,10 @@ de:
     notice_template_added: E-Mail-Vorlage wurde erfolgreich hinzugefügt.
     notice_template_destroyed: E-Mail-Vorlage wurde erfolgreich gelöscht.
     notice_template_updated: E-Mail-Vorlage wurde erfolgreich aktualisiert.
-    placeholders_hint: |
-      Die folgenden Platzhalter können jeweils im Abschnitt "Inhalt" verwendet werden
-      Vorlage.
+    placeholders_hint_header: |
+      Sie können die dynamische variable Unterstation in den Abschnitten `Betreff` und `Inhalt` verwenden
+      Vorlage. Vermeiden Sie die Verwendung von ereignisspezifischen Variablen, wenn Sie nicht möchten, dass Personen sie abrufen
+      mehr als eine E-Mail-Nachricht.
```
Comment (erdgeist) [inline config/locales/de.yml]: I don't get this translation. Was that auto generated from english?

File: config/locales/en.yml [hunk 2]
```diff
@@ -884,9 +884,10 @@ en:
     notice_template_added: Mail template was successfully added.
     notice_template_destroyed: Mail template was successfully destroyed.
     notice_template_updated: Mail template was successfully updated.
-    placeholders_hint: |
-      The following place holders may be used in the `content` section of each
-      template.
+    placeholders_hint_header: |
+      You can use dynamic variable substation in the in the `subject` and `content` sections of each
```
Comment (erdgeist) [inline config/locales/en.yml]: I don't get this translation. Was that auto generated from German?

Comment (erdgeist) [reply config/locales/en.yml]: Ahh, I get it. That should be a "substitution", not a substation.

Comment (eladeyal-intel) [reply config/locales/en.yml]: Fix (in all languages)

https://github.com/frab/frab/blob/24b078ab6c8e677983166143f42bb921330ed6a5/config/locales/en.yml#L887-L890

https://github.com/frab/frab/blob/24b078ab6c8e677983166143f42bb921330ed6a5/config/locales/de.yml#L899-L902

**Comment by eladeyal-intel** [comment]
fixed translations

**Pull request merged**